02/app.js

```diff
@@ -0,0 +1,105 @@
+const formEl = document.querySelector("form");
+const ulEl = document.createElement("ul");
+formEl.appendChild(ulEl);
+formEl.setAttribute("noValidate", true);
+const errors = [];
+
+formEl.addEventListener("submit", checkData);

```

Comment:
Raczej kod powinien "się czytać" od góry do dołu czyli na górze najważniejsze elementy czyli zmienne - i tutaj jest ok, ale potem deklaracja funkcji `checkData()` powinna być zaraz za addEventListnerem, poniżej funkcje, które są wykorzystywana bez pośrednie we wspomnianej funkcji itd.